### PR TITLE
fix(introspection): fix message from #6868 and comment warnings

### PR DESCRIPTION
**Pull request by Jolg42**
Now 
Screen Shot 2021-04-30 at 17 55 30

and --print
Screen Shot 2021-04-30 at 18 01 06

Seemed like a good idea to comment the warnings like the message

### CODE REVIEW

File: src/packages/migrate/src/commands/DbPull.ts [hunk 1]
```diff
@@ -354,6 +343,18 @@ continue with the instructions below.`
...
         .length
 
+      const prisma1UpgradeMessageBox = prisma1UpgradeMessage
+        ? '\n\n' +
+          drawBox({
+            height: 13,
+            width: 74,
+            str:
+              prisma1UpgradeMessage +
+              '\nOnce you upgraded your database schema to Prisma 2.0, \ncontinue with the instructions below.',
```
Comment (janpio) [inline src/packages/migrate/src/commands/DbPull.ts:353]: Empty line at end for beauty:
```suggestion
              '\nOnce you upgraded your database schema to Prisma 2.0, \ncontinue with the instructions below.\n',
```

**Review by janpio — CHANGES_REQUESTED** [review]
CLI output now seems to miss the headline!? (At least your screenshot indicates that)

**Comment by Jolg42** [comment]
@janpio good catch! fixed now 
Screen Shot 2021-04-30 at 18 13 40

**Review by janpio — APPROVED** [review]
Beautiful!

**Pull request merged**